src/tests/tests.fsx

```diff
@@ -0,0 +1,37 @@
+#r "System.Threading.dll"
```

Comment:
Sure, I just didn't want to remove it myself, in case you were incorporating it in a build somewhere. But yes, there should be only one, ideally, and it's a temporary measure until proper project file support appears.